src/crypto/convert.js

```diff
@@ -24,7 +24,18 @@ export const bufferToBigNumberString = bigNumberBuffer =>
 
 export const bufferToHex = buffer => naclInstance.to_hex(buffer);
 
-export const hexToBuffer = hex => Buffer.from(hex, 'hex');
+const hexRegex = /([0-9]|[a-f])/gim;
```

Comment:
Do we really want multiline?

Reply:
This regex gives odd behaviour.

```
> Buffer.from('abhcd', 'hex')
// Node 6: throws 'Invalid hex string'
<Buffer ab> // Node 8
> hexToBuffer('abhcd')
<Buffer abcd>
```

Reply:
will be changed to take only first part =)